<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetConfigureDialog</class>
 <widget class="QDialog" name="TilesetConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tileset Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/images/cat_builder/cat_builder_48.png</normaloff>:/images/cat_builder/cat_builder_48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Items</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="tilesetLayoutContainer">
     <item row="4" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Block</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BGO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NPC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scenery</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Path</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="spin_height">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spin_width">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QPushButton" name="clearTileset">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="SaveTileset">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tileset name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="OpenTileset">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_height">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <layout class="QVBoxLayout" name="tilesetLayoutWidgetContainer">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_width">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLineEdit" name="TilesetName"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="customOnly">
     <property name="text">
      <string>Show custom only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="specific">
     <property name="text">
      <string>Current Level/World specific</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="delete_me">
     <property name="text">
      <string>Delete this tileset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
